docs/book/v2/hot-code-reload.md

```diff
@@ -40,6 +40,14 @@ return [
 ];
 ```
 
+## Logging
+
+When a file is reloaded, a *notice* line will be logged with the message
+`File changed {path}, reloading`.
+
+The logger used to log these lines is the same used for access logging,
+which is described [here](logging.md#container-usage).
```

Comment:
This is definitely the correct format for the link, and our documentation renderer will correctly parse it.

Reply:
Right, the link has the correct format, but the text "here" is not very meaningful. Better would be something like: "which is described in the [Logging section](logging.md#container-usage) of this documentation"

(More informations in can be found in the W3C - QA Tips article: [Don't use "click here" as link text](https://www.w3.org/QA/Tips/noClickHere).)

Reply:
Agree on this